Makefile

```
@@ -84,6 +84,13 @@ build:
 install:
 	@go install -ldflags='$(LDFLAGS)' ./cmd/...
 
+docker:
+	DOCKER_BUILDKIT=1 docker build --pull --rm \
+		--build-arg LDFLAGS='$(LDFLAGS)' \
+  		-t "zqd:latest" -t "localhost:5000/zqd:latest" -t "localhost:5000/zqd:$(VERSION)" .
```

Delete the two spaces at the beginning of this line.
```suggestion
		-t "zqd:latest" -t "localhost:5000/zqd:latest" -t "localhost:5000/zqd:$(VERSION)" .
```